src/Tribe/JSON_LD/Event.php

```diff
@@ -17,7 +17,12 @@ class Tribe__Events__JSON_LD__Event extends Tribe__JSON_LD__Abstract {
 	 * @see https://developers.google.com/structured-data/rich-snippets/
 	 * @var string
 	 */
-	public $type = 'Event';
+	public    $type = 'Event';
+
+	/**
+	 * @var int
+	 */
+	protected $current_post_id;
```

Comment:
Question, couldn't apply the timezone on the `get_data` method? To avoid having to set and unset a filter plus a new propriety been set here?